04/index.html

```diff
@@ -1,16 +1,104 @@
 <!DOCTYPE html>
 <html lang="pl">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
 
     <link rel="stylesheet" href="./style.css" />
     <title>devmentor.pl - HTML & CSS: RWD - #04</title>
-</head>
-<body>
+  </head>
+  <body>
+    <nav class="nav" id="menu">
+      <label for="tm" id="toggle-menu" class="nav__toggle-menu"
+        >Open Menu
+        <span class="nav__open-hamburger"
+          ><img
+            src="./images/icon-hamburger.svg"
+            alt="Icon hamburger menu"
+            class="nav__drop-icon--hamburger" /></span
+      ></label>
+      <input type="checkbox" id="tm" />
+      <ul class="nav__main-menu clearfix">
+        <li><a href="#" class="nav__main-link">Start</a></li>
+        <li>
+          <a href="#" class="nav__main-link"
+            >Offer
+            <span class="nav__drop-icon">&#9662;</span>
+            <label title="Toggle Drop-down" class="nav__drop-icon" for="sm1"
+              >&#9662;</label
+            >
+          </a>
+          <input type="checkbox" id="sm1" />
+          <ul class="nav__sub-menu">
+            <li>
+              <a href="#" class="nav__sub-link"
+                >Websites
+                <span class="nav__drop-icon">&#9662;</span>
+                <label title="Toggle Drop-down" class="nav__drop-icon" for="sm3"
+                  >&#9662;</label
+                >
+              </a>
+              <input type="checkbox" id="sm3" />
+              <ul class="nav__sub-menu">
+                <li><a href="#" class="nav__sub-link">Websites 1.1</a></li>
+                <li><a href="#" class="nav__sub-link">Websites 2.1</a></li>
+              </ul>
+            </li>
 
-    <!-- navigation website -->
+            <li>
+              <a href="#" class="nav__sub-link"
+                >CMS
+                <span class="nav__drop-icon">&#9662;</span>
+                <label title="Toggle Drop-down" class="nav__drop-icon" for="sm2"
+                  >&#9662;</label
+                >
+              </a>
+              <input type="checkbox" id="sm2" />
```

Comment:
To rozwiązanie jest identyczne do tego przedstawionego w materiałach.
Kliknięcie w label (czyli strzałkę) powoduje zaznaczenie checkbox-a.
To wykorzystujesz w CSS, aby móc pokazywać lub ukrywać odpowiednie zagnieżdzenia.
W poprzednie rozwiązanie nie było złe. Wystarczyło `<img>` wyrzucić poza `<a>` wtedy można było wykorzystać `:hover`  z `@media` do pokazania zagnieżdżeń. W twoim rozwiązaniu kliknięcie w `<a>` czyli również w `<img> powodowało przejście do podstrony zamiast rozwinąć menu. Przypominam, że na urządzeniach mobilnych `:hover` działa na kliknięcie.